src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1045,12 +1045,45 @@ private void set_package_names_if_all_is_specified(ChocolateyConfiguration confi
                 config.Input = string.Empty;
 
                 var localPackages = list_run(config, logResults: false);
+                var packagesToUpdate = localPackages.Select(p => p.Key).ToList();

```

Comment:
Exactly: It was originally after `.Select`, which can never return a null `IEnumerable`, so I took the liberty of removing it.